tests/integration/test_app.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+from dotenv.main import find_dotenv, load_dotenv
+from todo_app.app import create_app
+from unittest.mock import Mock, patch
+import pytest
```

Comment:
You haven't added pytest or selenium to your pyproject.toml, so they won't get installed when people run `poetry install`.

You should have added them via `poetry add ...`. This will both install them and update pyproject.toml

To add them as dev only dependencies, you can run `poetry add pytest selenium --dev`